rtmbot.py

```diff
@@ -38,48 +42,54 @@ def start(self):
             self.output()
             self.autoping()
             time.sleep(.1)
+
     def autoping(self):
-        #hardcode the interval to 3 seconds
+        # hardcode the interval to 3 seconds
         now = int(time.time())
         if now > self.last_ping + 3:
             self.slack_client.server.ping()
             self.last_ping = now
+
     def input(self, data):
         if "type" in data:
             function_name = "process_" + data["type"]
             dbg("got {}".format(function_name))
             for plugin in self.bot_plugins:
                 plugin.register_jobs()
                 plugin.do(function_name, data)
+
     def output(self):
         for plugin in self.bot_plugins:
             limiter = False
             for output in plugin.do_output():
                 channel = self.slack_client.server.channels.find(output[0])
-                if channel != None and output[1] != None:
-                    if limiter == True:
+                if channel is not None and output[1] is not None:
+                    if limiter:
                         time.sleep(.1)
-                        limiter = False
-                    message = output[1].encode('ascii','ignore')
-                    channel.send_message("{}".format(message))
-                    limiter = True
+                    limiter = True  # TODO: check goal: no sleep for 1st channel, sleep of all after ?
+                    # TODO: find out how to safely encode stuff if needed :(
```

Comment:
Can you resubmit the PR without changing these lines around the message sending?